www/src/components/Menu.jsx

```diff
@@ -96,6 +98,11 @@ const MenuComponentListCategory = ({ children, title }) => (
   </div>
 );
 
+const handlePlaygroundClick = () => {
+  global.analytics.track('Visit playground',
+    { event: 'openedx.paragon.menu.playground.visit_playground.clicked' });
```

Comment:
`openedx.paragon.menu.playground.visit_playground.clicked` is intended to be the actual event name, not an event property here. Generally, I'd like us to standardize on this format for event names.

```js
global.analytics.track('openedx.paragon.menu.playground.visit_playground.clicked');
```